contributing/labeling.md

```diff
@@ -0,0 +1,15 @@
+# Label Definitions
+
+* `Epic`: Big issue to be broken down into smaller tasks
+
+### Priority (`p.*`)
```

Comment:
`High/Low` would suffice imo. We should use the `Milestone` links which already indicates that an issue has a higher relevancy than those that do not have a milestone attached to it. `High` prio as an additional label might only be necessary for bugs.